ci/shellcheck.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright (c) 2007-2018 The OpenRC Authors.
+# See the Authors file at the top-level directory of this distribution and
+# https://github.com/OpenRC/openrc/blob/master/AUTHORS
+#
+# This file is part of OpenRC. It is subject to the license terms in
+# the LICENSE file found in the top-level directory of this
+# distribution and at https://github.com/OpenRC/openrc/blob/master/LICENSE
+# This file may not be copied, modified, propagated, or distributed
+# except according to the terms contained in the LICENSE file.
+
+set -e
+set -u
+
+shellcheck --version
+for shellscript in $(git ls-files); do
+	case $shellscript in
+		init.d/*.in) ;;
+		sh/*.in) ;;
+		sh/*.sh) ;;
+		support/init.d.examples/*.in) ;;
+		support/openvpn/*.sh) ;;
+		support/sysvinit/halt.sh) ;;
+		test/*.sh) ;;
+		*) continue ;;
+	esac
+	echo "Checking ${shellscript} with shellcheck:"
+	shellcheck -s sh "${shellscript}" || true
```

Comment:
this ignores all errors, so this job will never fail.  since it will always be green (in the actions tab, and in PRs), no one will ever look at it.  so like, what's the point ? :)

i think you want to set `ret=0` at the top of the file, change this `true` to `ret=1`, and then `exit ${ret}` at the bottom.

Reply:
none of the CI checks block merges or pushes, so i think it'd be OK to just land this as-is.  but i'm fine with an `exit 0` at the end of the script with a `TODO` explaining why it's ignoring the `$ret` if you want to go that route.